### PR TITLE
Validate pupcet

### PR DESCRIPTION
**Pull request by pupcet**
# Validate [pupcet](https://solscan.io/token/3peLCS3GqeWHoAN9wi9hvzY46Rg3695DW3vkNM37sP8f)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/pupcet_solana/status/1768344565914767842
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/YOUR_ID_HERE

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [ ] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Comment by pupcet** [comment]
Community tweet: https://twitter.com/pupcet_solana/status/1768344565914767842

**Comment by vortexvanguard000** [comment]
It was community takeover so we would appreciate if you list us 💪

File: validated-tokens.csv [hunk 1]
```diff
@@ -1183,3 +1183,4 @@ JENSEN HUANG,JENSEN,HfQuaLjMguh7vyZqqrWuVqpsjQd7tAPrzBWKqA3pDTH3,9,https://i.ibb
 Stanley Cup Coin,STAN,CQSzJzwW5H1oyWrp6QhfUKYYwyovbSiVDKnAxNfb1tJC,5,https://arweave.net/urnvj3NcnKmwEWNLDQJetp2OFP_nu3tB8HPdXwSFg7U,true
 RARE,RARE,HmLspvjpQtQEnArUyJoBSFGS38gNJwBuxAeqSV9SZ66K,9,https://raw.githubusercontent.com/SuperRareBears/srb-assets/main/tokens/rare/logo.png,true
 ROD.AI,RODAI,GdbyLsNKHKLXTZVEo8QrGKVmrexEeZUYvhpSfuZ9TdkC,5,https://www.rodai.meme/logo.png,true
+Pupecet,pupcet,3peLCS3GqeWHoAN9wi9hvzY46Rg3695DW3vkNM37sP8f,6,https://bafkreid35bvrzo76g4am7pbp4lyyrexjgg6a7thxz3lwdz4uetclumctw4.ipfs.nftstorage.link/,true
```
Comment (vortexvanguard000) [inline validated-tokens.csv:1186]: It was community takeover so we would appreciate if you list us 💪

**Review by vortexvanguard000 — COMMENTED** [review]
We will make biggest comeback

**Review by vortexvanguard000 — COMMENTED** [review]
It was community takeover so we would appreciate if you list us 💪

File: validated-tokens.csv (hunk 1 above)
Comment (embereclipse3) [inline validated-tokens.csv:1186]: New cat in town!!!

Comment (embereclipse3) [inline validated-tokens.csv:1186]: https://github.com/jup-ag/token-list/assets/163508749/4b46a63f-4956-48ed-8c0e-1c63f53c7540

**Review by embereclipse3 — COMMENTED** [review]
New cat in town!!!

**Review by embereclipse3 — COMMENTED** [review]
https://github.com/jup-ag/token-list/assets/163508749/c9afb5b0-9cfa-44dd-8447-6ef1fa540196

**Comment by Jymfletzfur** [comment]
insufficient mcap

**Pull request closed without merging**